CHANGELOG.md

```diff
@@ -6,23 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Features
+## [v2.0.0-beta1](https://github.com/regen-network/regen-ledger/releases/tag/v2.0.0-beta1) - 2021-08-18
 
-* Add support for on-chain creation of Permanent Locked Accounts (#482, [regen-network/cosmos-sdk#42](http://github.com/regen-network/cosmos-sdk/pull/42)
+### General
+
+#### Added
+
+* [#388](https://github.com/regen-network/regen-ledger/pull/388) Add support for rosetta
+* ([#482](https://github.com/regen-network/regen-ledger/pull/482),
+    [regen-network/cosmos-sdk#42](http://github.com/regen-network/cosmos-sdk/pull/42))
+    Add support for on-chain creation of Permanent Locked Accounts
+* [#349](https://github.com/regen-network/regen-ledger/pull/349) Add x/feegrant & x/authz from Cosmos SDK v0.43
+
+#### Changed
+
+* [#422](https://github.com/regen-network/regen-ledger/pull/422) remove `Request` suffix in Msgs
+* [#322](https://github.com/regen-network/regen-ledger/pull/322) Split regen ledger into multiple go modules
+* [#482](https://github.com/regen-network/regen-ledger/pull/482) Upgrade Cosmos SDK to v0.43.0
+
+#### Fixed
+
+* [#386](https://github.com/regen-network/regen-ledger/pull/386) fix IBC proposal registration
 
 ### `x/ecocredit`
 
 #### Added
-* (genesis) [#389](https://github.com/regen-network/regen-ledger/pull/389) add genesis import and export.
-* add support for credit cancelling (#385)
-* record retirement locations of ecocredit (#328)
-* add dates as top level fields in credit batches (#393)
-* add project location as field in credit batches (#394)
-* use dec wrapper for decimal operations (#435)
-* add credit types to credit class (#424)
-* add params for an allowlist of permissioned credit designers (#425)
-* add queries to list classes and batches with a class (#451)
-* update class ID and batch denomination formats (#392)
+
+* (genesis) [#389](https://github.com/regen-network/regen-ledger/pull/389) add genesis import and export
+* [#385](https://github.com/regen-network/regen-ledger/pull/385) add support for credit cancelling
+* [#425](https://github.com/regen-network/regen-ledger/pull/425) add params for an allowlist of permissioned credit designers
+* [#451](https://github.com/regen-network/regen-ledger/pull/451) add queries to list classes and batches with a class
```

Comment:
These three are issues rather than pull requests. I think it's ok to link the issue but I just thought I might point that out. You can use `/issue/` or `/pull/` and the links resolve either way.

Reply:
Yeah I think in general we should probably be sticking w/ PRs. Saw that some issues snuck in but I was a bit lazy to check them in detail. Let's try and keep practice of using PRs in changelog moving forward.